kong/conf_loader/init.lua

```diff
@@ -1856,6 +1848,53 @@ local function load(path, custom_conf, opts)
     end
   end
 
+  -- Wasm module support
+  if conf.wasm then
+    local wasm_filters = get_wasm_filters(conf.wasm_filters_path)
+    conf.wasm_modules_parsed = setmetatable(wasm_filters, _nop_tostring_mt)
+
+    local function add_wasm_directive(directive, value, prefix)
+      local directive_name = (prefix or "") .. directive
+      conf[directive_name] = conf[directive_name] or value
```

Comment:
Question: does this need to be an explicit `nil` check rather than just false-y? It's hard for me to quickly grok the paths through the conf_loader, so I'm not sure if we expect only string values at this point.

```suggestion
      if conf[directive_name] == nil then
        conf[directive_name] = value
      end
```

Reply:
That is a great catch. I've also added one additional test case - to ensure explicitly set wasm directives take precedence.